streams/src/test/java/org/apache/kafka/streams/processor/internals/TaskManagerTest.java

```diff
@@ -1578,6 +1574,10 @@ public void shouldRethrowTaskCorruptedExceptionFromStateUpdater() {
         assertEquals(mkSet(taskId00, taskId01), thrown.corruptedTasks());
         assertEquals("Tasks [0_1, 0_0] are corrupted and hence need to be re-initialized", thrown.getMessage());
     }
+
+    // this @Test annotation was removed via https://github.com/apache/kafka/pull/15896
+    // but I am not sure why -- maybe by mistake?
+    @Test
```

Comment:
Does anybody know? @cadonna maybe?

Reply:
I am quite sure, it was a mistake that we missed.